src/services/workshops/hooks.js

```diff
@@ -19,17 +19,40 @@ async function assignToOwner({ params: { user: owner }, app, result, service })
 async function checkXMLChanged(context) {
     const workshop = await context.app.service('workshops').get(context.id);
 
-    if (context.data.hasOwnProperty('xml')) {
+    if (context.data.hasOwnProperty('xml') && !context.data.hasOwnProperty('properties')) {
         workshop.xml !== context.data.xml ?
             context.data.xmlChanged = true :
             context.data.xmlChanged = false;
-    } else {
+    }
+
+    else if (!context.data.hasOwnProperty('xml') && context.data.hasOwnProperty('properties')) {
+        propertiesChanged(workshop, context) ?
```

Comment:
Please avoid side effects in ternaries. This would be much better:
```javascript
context.data.xmlChanged = propertiesChanged(workshop, context);
```